bench/algorithm/coro-prime-sieve/1.dart

```diff
@@ -1,27 +1,34 @@
 import 'dart:async';
 
 Future main(List<String> arguments) async {
-  final n = arguments.length > 0 ? int.parse(arguments[0]) : 5;
+  final n = arguments.isNotEmpty ? int.parse(arguments[0]) : 5;
+
   var stream = StreamIterator(generate());
+
   for (var i = 0; i < n; i++) {
```

Comment:
for (var i = 0; i < n && await stream.moveNext(); i++) {
    var prime = stream.current;
    print(prime);
    stream = StreamIterator(filter(stream, prime));
  }